ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -142,6 +149,22 @@ public struct MSFButtonView: View {
                    theme: theme)
     }
 
+    // MARK: - UIGestureRecognizerDelegate
+
+    public func gestureRecognizer(_ gestureRecognizer: UIGestureRecognizer,
+                                  shouldRecognizeSimultaneouslyWith otherGestureRecognizer: UIGestureRecognizer) -> Bool {
+        // Prevents the Large Content Viewer gesture recognizer from cancelling the SwiftUI button gesture recognizers.
+        return true
```

Comment:
add apple radar bug number to remove? also, how does pointer interaction work with this now?

Reply:
also it could return false if content viewer isn't needed?

Reply:
I'll add the Apple radar bug number once we have it created. I'll follow up on the process.
As far as pointer interaction support, none of the FluentUI Buttons (UIKit or Vnext) currently support iPad pointer interaction. We can work on that as a new work item.